vue3/package.json

```diff
@@ -35,5 +35,6 @@
     "vite": "^5.4.7",
     "vite-plugin-vuetify": "^2.0.4",
     "vue-tsc": "^2.0.26"
-  }
+  },
+  "packageManager": "yarn@1.22.22"
```

Comment:
what version of `yarn` do you want to support? it feels like this should be a monorepo, since `vue3` is referencing `vue`, which would require upwards of yarn v1 (current latest at `yarn@4.7.0`). even if not wanting to use yarn workspaces, upgrading from v1 would give more reliability/performance surrounding node modules

Reply:
sure, feel free to upgrade. As with node I have never tought about this version. The `vue` folder will disappear at some point, just left it in because the old stuff is not yet cleaned out.